Neos.Eel/Configuration/Objects.yaml

```diff
@@ -21,6 +22,16 @@ Neos\Eel\CompilingEvaluator:
 
 Neos\Eel\EelEvaluatorInterface:
   className: Neos\Eel\CompilingEvaluator
+  # FIXME: This configuration shouldn't be necessary. Check in ObjectConfiguration.
```

Comment:
Well, the interface does not describe the constructor, could that be the reason?

Also, if doing this for the interface, the constructor would _have_ to be included there, otherwise this would not work if the implementation deviates. Which is may well do, unless the constructor is in the interface. Which it isn't.

You see? ;)

Reply:
Right, very good point. I will move it to setter.